docs/modules/README.md

```diff
@@ -12,9 +12,11 @@ ESLint plugin.
 
 ## List of modules
 
+- [`aggregate-error`](./aggregate-error.md)
 - [`bluebird`](./bluebird.md)
 - [`cpx`](./cpx.md)
 - [`deep-equal`](./deep-equal.md)
+- [`es-aggregate-error`](./es-aggregate-error.md)
```

Comment:
Since it's a polyfill, I'm not sure that it requires its own docs page.

Reply:
it may be that we shouldn't add these to the docs 🤔 

since its a native feature with an MDN doc

what do you think?